dotnet/src/SemanticKernel.Abstractions/Diagnostics/SKException.cs

```diff
@@ -7,20 +7,20 @@ namespace Microsoft.SemanticKernel.Diagnostics;
 /// <summary>
 /// Provides the base exception from which all Semantic Kernel exceptions derive.
 /// </summary>
-public abstract class SKException : Exception
+public class SKException : Exception
```

Comment:
Yep, it was discussed during the ADR review. The decision was not to do that because .Net exception design guidelines say not to that - https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/using-standard-exception-types